src/core/lib/surface/call.cc

```diff
@@ -415,18 +415,17 @@ class ChannelBasedCall : public Call {
 
   void DeleteThis() {
     RefCountedPtr<Channel> channel = std::move(channel_);
-    Arena* arena = arena_;
+    RefCountedPtr<Arena> arena = arena_;
```

Comment:
`std::move()`?  Or does something in the dtor need `arena_` to still be set?

Reply:
anything that accesses contexts.... it's a bit unsafe to clear it